index.html

```diff
@@ -1672,6 +1672,28 @@ <h3>Create MD5 checksums (audio samples)</h3>
     </div>
     <!-- ends Create frame md5s (audio) -->
 
+    <!-- Create stream md5s -->
+    <label class="recipe" for="create_stream_md5s">Create MD5 checksum(s) for A/V stream data only</label>
+    <input type="checkbox" id="create_stream_md5s">
+    <div class="hiding">
+      <h3>Create stream MD5s</h3>
+      <p><code>ffmpeg -i <i>input_file</i> -map 0:v:0 -c:v copy -f md5 <i>output_file1</i> -map 0:a:0 -c:a copy -f md5 <i>output_file2</i></code></p>
```

Comment:
Nit: I would write `output_file_1` and `output_file_2` for readability.

Reply:
yeah, that makes sense!